kubernetes/cmsweb/services/newdqmgui.yaml

```diff
@@ -1,4 +1,24 @@
 ---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-cvmfs-cms
+provisioner: cvmfs.csi.cern.ch
+parameters:
+  repository: cms.cern.ch
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: csi-cvmfs-cms-pvc
+spec:
+  accessModes:
+  - ReadOnlyMany
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: csi-cvmfs-cms
+---
```

Comment:
I have included above section in another file that can be used in multiple deployments https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/storages/dqm-cvmfs.yaml
You may remove above section from here

Reply:
I removed the section